tests/smoke/features/step_definitions/api.ts

```diff
@@ -15,11 +15,25 @@ Given(
 )
 
 Given(
-  'admin set the default folder for received shares to {string}',
+  'set default share folder to {string}',
```

Comment:
If you don't mind I'd go ahead and push some wording changes to the step definition on this PR? 🤓 😇

Reply:
yes, of course)